modules/invoicenote.php

```diff
@@ -89,6 +89,7 @@
     $cnote['sdate'] = $invoice['sdate'];
     $cnote['customerid'] = $invoice['customerid'];
     $cnote['reason'] = '';
+    $cnote['numberplanid'] = $invoice['numberplanid'];
```

Comment:
Bzdura - jak możesz kopiować plan z faktury do korekty?

Reply:
Przecież dla korekt i dla faktur jest ten sam formularz... popatrz szerzej niż tylko ta linia.

Reply:
Przetestowałem to i po commicie wymieniony błąd w zgłoszeniu nie występuje.

![testowy-jan](https://user-images.githubusercontent.com/17087236/120011225-5a167c00-bfde-11eb-9e8f-e9c9c1974425.png)